com.github.sakya.corechess.json

```diff
@@ -0,0 +1,64 @@
+{
+  "app-id": "com.github.sakya.corechess",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "21.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.dotnet6"
+  ],
+  "command": "CoreChess",
+  "finish-args": [
+    "--filesystem=host",
```

Comment:
why `host` filesystem access for a chess games. at best `xdg-documents` for saving games.

Reply:
Because this is just a gui, it doesn't include any engine. 
Engines must be installed somewhere in the host system

Reply:
this won't work. You can't run executable from the distribution.

You should provide the engines in the flatpak.

Reply:
Is this some limitation specific for distributing the package via falthub? 
The package runs fine on my system when testing it and it executes the engine form my home.

Distributing engines in the flatpak isn't really good: there are a lot of UCI engines. It would mean the user can only use the engines I decide to include.

Reply:
Flatpak application runs sanboxed. It's not designed like distro packages with dependency hell. Because it is sandboxed, it doesn't allow runing binaries from the host. 

If you feel engines are too big there is a way to package them as extensions so the user can install them, at the detriment of easiness.

Reply:
I don't understand why it works if I build, install and run the flatpak on my system?
And how does something like Visual Studio Code can run a compiler or interact with docker (I never used vscode installed via flatpak)?

Reply:
I think I can modify my applications and let the user downloads some engines after the installation.
This way I can completely remove the "host" permission (I'd have to add internet access btw).
Many thanks for your help.
Should I close this pull request or let it open until I modify my application?

Reply:
I do not recommend this approach.

Reply:
Why? What's the problem if the user downloads an engine into the sandboxed filesystem (in .var/app/com.github.sakya.corechess/data/CoreChess/ for example) and the run it from there?